### ISSUE TITLE
When I use the openSUSE Tumbleweed booted by it, there are multiple rpm package verification errors.

### ISSUE THREAD

**Issue by ixnij** [ventoy/Ventoy#914]
I cannot provide specific log files, but the iso file I used was downloaded from https://software.opensuse.org/distributions/tumbleweed, and the iso after downloading has been verified and has not been tampered with.

**Comment by ixnij** [comment]
And, as far as I know, many openSUSE users have the same problems as me.

**Comment by ventoy** [comment]
Any photos please?
What should I do after boot to trigger the rpm package verification errors?

**Comment by ixnij** [comment]
> Any photos please?
> What should I do after boot to trigger the rpm package verification errors?

When installing the system, the installer of openSUSE will verify each software package to check the integrity of the package. My problem arises when installing the system: The installer of openSUSE is installed one by one, and the verification work This is also done one after another, the installer keeps popping up messages that the package has not passed the integrity check, so that I cannot install the system.

**Comment by ventoy** [comment]
Did you checksum the iso file in Ventoy's menu (not in the OS) as follows: ?
https://www.ventoy.net/en/faq.html#faq_boot_checksum

**Comment by ixnij** [comment]
> Did you checksum the iso file in Ventoy's menu (not in the OS) as follows: ?
> https://www.ventoy.net/en/faq.html#faq_boot_checksum

No,I didn't. Maybe this is why my problem arises. I will try to verify my iso file in ventoy. Thanks!

**Comment by ventoy** [comment]
I test this iso and it works fine (installation successfully finished).
![image](https://user-images.githubusercontent.com/59477474/119223967-cd2b7880-bb2e-11eb-9761-f03dc7bfb75e.png)

**Comment by ixnij** [comment]
> I test this iso and it works fine (installation successfully finished).
> ![image](https://user-images.githubusercontent.com/59477474/119223967-cd2b7880-bb2e-11eb-9761-f03dc7bfb75e.png)

Did you install it successfully on your real machine instead of qemu or virtualbox? I think their behavior is different on real computers and virtual machines.

**Comment by ventoy** [comment]
@ixnij  What about the latest Ventoy release?
